src/main/java/org/gridsuite/network/map/dto/utils/ElementUtils.java

```diff
@@ -62,9 +62,66 @@ public static ConnectablePositionInfos toMapConnectablePosition(Identifiable<?>
         return builder.build();
     }
 
-    public static CurrentLimitsData toMapDataCurrentLimits(Branch<?> branch, Branch.Side side) {
-        Optional<CurrentLimits> limits = side == Branch.Side.ONE ? branch.getCurrentLimits1() : branch.getCurrentLimits2();
-        return limits.map(l -> toMapDataCurrentLimits(l)).orElse(null);
+    public static Optional<ActivePowerControlInfos> toActivePowerControl(Identifiable<?> identifiable) {
```

Comment:
return activePowerControl == null ? Optional.empty() :
            Optional.of(ActivePowerControlInfos.builder()
                .activePowerControlOn(activePowerControl.isParticipate())
                .droop(activePowerControl.getDroop()).build());

Reply:
Done